### ISSUE TITLE
module 'os' has no attribute 'startfile'

### ISSUE THREAD

**Issue by nurb2kea** [atticus-lv/aces-helper#2]
Hi,

blender 3.1 macos monterey

Python: Traceback (most recent call last):
  File "/Applications/BLENDER/ADDONS etc/addons/aces-helper-main/__init__.py", line 107, in execute
    os.startfile(self.path)
AttributeError: module 'os' has no attribute 'startfile'

location: <unknown location>:-1

**Comment by atticus-lv** [comment]
Since I don't have a mac computer, I need someone to help develop it

**Comment by nurb2kea** [comment]
Sounds good to me.
But must not be specific for mac/win/linux.
Anyways, nice helper.

Thank you :-)

**Comment by atticus-lv** [comment]
> Sounds good to me. But must not be specific for mac/win/linux. Anyways, nice helper.
> 
> Thank you :-)

Just add a new commit 0a5d7212d6e1ae42ac84624cfb741f9e9a77bd97
Can you  help me the test it? Just down the latest code. It may work on mac.

**Comment by nurb2kea** [comment]
I'm kinda beta tester for some things. So no problem. :-)

![Screen Shot 2021-12-02 at 11 54 39 pm](https://user-images.githubusercontent.com/57676085/144516182-6c6a2a0c-054b-40d5-bd01-56da1c146141.jpg)

**Comment by nurb2kea** [comment]
Found this about mac in another script, maybe helps...

def execute(self, context):
        try:
            if bpy.app.build_platform == b'Windows':
                subprocess.run(["start", "", bpy.app.binary_path], shell=True)
            elif bpy.app.build_platform == b'Linux':
                subprocess.run(["gnome-terminal", "-x", "sh", "-c", bpy.app.binary_path, "&", "disown"])
            elif bpy.app.build_platform == b'Darwin':
                subprocess.run(["open", "-n", "-a", bpy.app.binary_path])
            else:
                subprocess.run([bpy.app.binary_path])
        except:
            print("Unable to create a new blender instance")

        return {'FINISHED'}

**Comment by nurb2kea** [comment]
Another example about darwin:

# Open Explorer where rendered images were saved
        if _platform == "darwin":
            bat.write('open ' + '\"' + opath  + '\" \n')
            bat.write('read -p "Press Enter to close the Terminal..."')
        elif _platform == "linux" or _platform == "linux2":
            bat.write('xdg-open ' + '\"' + opath  + '\" \n')
            bat.write('read -p "Press Enter to close the Terminal..."')
        elif _platform == "win32" or _platform == "win64":
            bat.write('start \"Render Output\" ' + '\"' + opath  + '\" \n')
            bat.write('pause')
        bat.close()

**Comment by nurb2kea** [comment]
No way I get this fixed myself !

**Comment by atticus-lv** [comment]
Wow. Nice! Can you pull a request to commit it ? I am too busy these day so I have no time to cover it

**Comment by nurb2kea** [comment]
pull a request to commit it... ???
No idea what do you mean by that.

**Comment by atticus-lv** [comment]
emmm, It mean contribute your code(what make it work on mac) to this repostory.
First step , fork from this repostory.
![image](https://user-images.githubusercontent.com/64118610/144728945-fb683e41-2205-4e92-b687-4f598a8e97d3.png)
Second, select the file you want to change in your repostory and edit it(Save is to commit it to your own repostory)
![image](https://user-images.githubusercontent.com/64118610/144728966-e2676454-1081-4b29-8fae-e6b1f51984fc.png)
Third, back to the main page of your repostory and choose to contribute the changes(commit) to my repostory(call pull request)
![image](https://user-images.githubusercontent.com/64118610/144728997-7bc01c50-42a7-4077-a6df-5b642e1b2ad7.png)

**Comment by nurb2kea** [comment]
I have no clue about coding in py, I'm mostly copy and paste code snippets in the hope it could work.
But I have no idea how to just open the finder to choose a path on mac. (That was what I tried to copy out of other scripts to give you an idea.)

Anyways, I downsized my ACES colormanagement lut folder and the list of profiles in the meantime instead.
Changed input method for color picker...
So I eded up with the important profiles short list that can be handled like the usual profile choosing in vanilla blender.

Makes this addon almost useless.
(No need to click on the node beforehand to activate the right click menu on the node, to choose a profile)
One addon less for me.

Anyways, even you couldn't solve it, thanks for the help
